src/main/java/uk/gov/hmcts/reform/cwrdapi/config/MessagingConfig.java

```diff
@@ -1,66 +1,46 @@
-import org.springframework.jms.core.JmsTemplate;
-import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
-import org.springframework.jms.support.converter.MessageConverter;
-import org.springframework.jms.support.converter.MessageType;
+import org.springframework.stereotype.Service;
 
-import javax.jms.ConnectionFactory;
-import javax.net.ssl.SSLContext;
-
-@Configuration
+@Service
 @Slf4j
 public class MessagingConfig {
 
+
     @Value("${loggingComponentName}")
     private String loggingComponentName;
 
-    @Bean
-    public String jmsUrlString(@Value("${crd.publisher.azure.service.bus.host}") final String host) {
-        return String.format("amqps://%1s?amqp.idleTimeout=3600000", host);
-    }
+    @Value("${crd.publisher.caseWorkerDataPerMessage}")
+    private int caseWorkerDataPerMessage;
 
-    @Bean
-    public ConnectionFactory jmsConnectionFactory(
-            @Value("${spring.application.name}") final String clientId,
-            @Value("${crd.publisher.azure.service.bus.username}") final String username,
-            @Value("${crd.publisher.azure.service.bus.password}") final String password,
-            @Autowired final String jmsUrlString,
-            @Autowired(required = false) final SSLContext jmsSslContext,
-            @Value("${crd.publisher.azure.service.bus.trustAllCerts}") final boolean trustAllCerts) {
+    @Value("${crd.publisher.azure.service.bus.host}")
+    String host;
 
-        JmsConnectionFactory jmsConnectionFactory = new JmsConnectionFactory(jmsUrlString);
-        jmsConnectionFactory.setUsername(username);
-        jmsConnectionFactory.setPassword(password);
-        jmsConnectionFactory.setClientID(clientId);
-        jmsConnectionFactory.setReceiveLocalOnly(true);
-        if (trustAllCerts && jmsSslContext != null) {
-            jmsConnectionFactory.setSslContext(jmsSslContext);
-        }
-        return new CachingConnectionFactory(jmsConnectionFactory);
-    }
+    @Value("${crd.publisher.azure.service.bus.topic}")
+    String topic;
 
-    @Bean
-    public JmsTemplate jmsTemplate(ConnectionFactory jmsConnectionFactory) {
-        JmsTemplate returnValue = new JmsTemplate();
-        returnValue.setConnectionFactory(jmsConnectionFactory);
-        returnValue.setMessageConverter(new MappingJackson2MessageConverter());
-        returnValue.setSessionTransacted(true);
-        return returnValue;
-    }
+    @Value("${crd.publisher.azure.service.bus.username}")
+    String sharedAccessKeyName;
+
+    @Value("${crd.publisher.azure.service.bus.password}")
+    String sharedAccessKeyValue;
 
-    @Bean // Serialize message content to json using TextMessage
-    public MessageConverter jacksonJmsMessageConverter() {
-        MappingJackson2MessageConverter converter = new MappingJackson2MessageConverter();
-        converter.setTargetType(MessageType.TEXT);
-        return converter;
+    public ServiceBusSenderClient getServiceBusSenderClient() {
```

Comment:
you can create a bean for it by just appending `@Bean` on top of it

Reply:
Regarding the autowiring, we have deliberately avoided creating the bean.
We would like to create a fresh connection on every file upload, thus avoiding problem of long running connections.
@shashank-rastogi Can you please share your thoughts as well.

Reply:
Hi @SujeethKumarAri @abhijitdiwan I have created the bean for service bus client. Can you please have a look.
Thanks